portfolio/src/main/webapp/hobbies.html

```diff
@@ -0,0 +1,87 @@
+                at local college events. But actually, shifting my focus more to academics has not diminished my appreciation for music but actually
+                enhanced it! This semester in my world literature class, we read two works that have actually inspired two of the most famous
+                musical pieces in the world: Arabian Nights, which inspired Korsakov's Scheherezade, and Candide, which of course, inspired Bernstein's Candide Overture. 
+                While reading those pieces, I was able to gain a new appreciation for the symphonies I had played before, drawing connections between the story and 
+                the intentions of the composer in establishing certains themes and tones throughout the piece. In the same vein, having a recollection of the music 
+                enhances the way I interact with the literature, which has been really rewarding.</p>
+        </div>
+        <div id="htr">
+            <img class="floated" src="images/HTR.png">
+            <h2>Harvard Technology Review</h2>
+            <p> One of the experiences I have really enjoyed in college is writing for the Harvard Technology Review. Namely, 
+                I enjoy interviewing faculty members from Harvard/MIT and conversing with them about their work and ideas. For my first piece,
+                I interviewed Sarah Newman and we discussed ideas at the intersection of AI and Agency. For my second piece, I took the T down to Beth Semel at MIT where 
+                we talked about the intricacies of computational psychiatry and the impact of technology on the healthcare system. And most recently, I interviewed Vijay Iyer, 
+                a grammy award-winning jazz pianist, electronic musician, and Harvard music faculty member on the intersection of technology and music. Every opportunity to talk to sometime who has devoted their entire life
+                to a particular niche field within technology has been humbling and inspiring and I hope to continue to write and share their ideas with the world during
+                my time at Harvard.</p>
+        </div>
+        <div id="reading-group">    
+            <img class="floated" src="images/Paxos.png">
+            <h2>Reading Group</h2>
+            <p> A couple of my friends who are absolute nerds started this computer science and statistics reading group. Every week,
+                a group of students comes together to discuss topics from two different related research papers. 
+                It's honestly really nice to get exposed to different topics within CS/Stat and 
+                to be able to learn about practical applications of the material we learn in class.
+                Thus far, we have discussed papers on differential privacy, a field spearheaded by Harvard's very own 
+                Cynthia Dwork, and this week we will be looking into the PAC algorithm and Google Spanner, which is
+                ironically close to home.</p>
```

Comment:
Really nice paragraphs :) They were lovely to read

Reply:
thank you :)